tests/unit/amo/pages/TestCollection.js

```diff
@@ -771,6 +771,13 @@ describe(__filename, () => {
     });
   });
 
+  it('does not render a CollectionControls component when creating', () => {
+    const { store } = dispatchSignInActions();
+    const wrapper = renderComponent({ creating: true, store });
+
+    expect(wrapper.find(CollectionControls)).toHaveLength(0);
```

Comment:
I think you could move this assertion in the test case below (around L840)

`it('renders a create collection page', ...);`

Reply:
Ok, thanks.